traffic_portal/app/src/modules/private/user/UserController.js

```diff
@@ -51,6 +51,9 @@ var UserController = function($scope, $state, $location, $uibModal, formUtils, l
     $scope.user = userModel.user;
 
     $scope.confirmSave = function(user, usernameField) {
+        if (usernameField == undefined) {
```

Comment:
comparisons should use `===` - under what conditions was this occurring?

Reply:
This happens when the user tries to Update the password after clicking on the `Reset Password` link in TP